agent/01_agent_requirements.sh

```diff
@@ -63,8 +63,6 @@ fi
 
 if [[ "${AGENT_E2E_TEST_BOOT_MODE}" == "ISO_NO_REGISTRY" ]]; then
    sudo dnf -y install xorriso coreos-installer syslinux skopeo
-    
-   early_deploy_validation
```

Comment:
Already called from https://github.com/openshift-metal3/dev-scripts/pull/1755/commits/f11cb0d2cfc437a6b21822da280664f6ebe5d943#diff-054af3cea588fc3c166a66dbc56c926ab31695db4f6d47d2af638f759303106dR15

Reply:
Seems reasonable to remove this duplicate call.